src/App/App.tsx

```diff
@@ -1,25 +1,32 @@
 import { Text, Title } from '@mantine/core';
 import { ReactLocation, Router } from '@tanstack/react-location';
 
+import { AppShell } from '../components/appShell';
 import { useGetAdminListQuery } from '../operations';
 
 const location = new ReactLocation();
 
-export const App: React.FC = () => {
+export const App = () => {
   const adminListQueryResult = useGetAdminListQuery();
   return (
     <Router routes={[{ path: '/' }]} location={location}>
-      <Title order={1}>The Charity App</Title>
-      <br />
-      <Title order={3}>
-        {adminListQueryResult.loading
-          ? 'loading data'
-          : adminListQueryResult.error != null
-          ? 'error occurred while loading admins ids'
-          : adminListQueryResult.data?.admins.map(({ id }) => (
-              <Text key={id}>admin with id: {id}</Text>
-            ))}
-      </Title>
+      <AppShell
+        content={
+          <>
+            <Title order={1}>The Charity App</Title>
+            <br />
+            <Title order={3}>
+              {adminListQueryResult.loading
+                ? 'loading data'
+                : adminListQueryResult.error != null
+                ? 'error occurred while loading admins ids'
+                : adminListQueryResult.data?.admins.map(({ id }) => (
+                    <Text key={id}>admin with id: {id}</Text>
+                  ))}
+            </Title>
+          </>
+        }
+      />
```

Comment:
we have to separate the `Home` from the `App` which handles the routing and then put the appshell inside the `Home` component

maybe after merging this I can fix it inside the init btn pr cause I've created the Home component there